docs/section9/Rule6-ja.md

```diff
@@ -16,28 +16,42 @@
 **Data Lookup:** Table G3.7 and Table G3.8  
 ## Rule Logic: 
 
-- Get the building area lighting area type of the building segment in the Proposed model: ```building_light_area_type = building_segment.lighting_building_area_type``` (Note XC, assuming this field lists values from T-9.5.1 or T-G3.8, for "Building Area Method", or "None" if the building goes for "Space-by-Space Method")
+- For each building_segment in the Proposed Model: ```building_segment_proposed in P_RMR.building.building_segments```  
 
-- Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+- Get the lighting_building_area_type for the building_segment: ```lighting_building_area_type_proposed = building_segment_proposed.lighting_building_area_type```  
 
-- Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
+  - Determine if the building segment uses Building Area Method: ```if lighting_building_area_type_proposed is "None": bam_flag = FALSE else bam_flag = TRUE```  
```

Comment:
Lines 21 and 23 could be combined by replacing `lighting_building_area_type_proposed` in line 23 with `buliding_segment_proposed.lighting_buildling_area_type`.